DEVELOPMENT.md

```diff
@@ -77,15 +77,30 @@ real life testing.
 
 ## Publishing a new kured release
 
-This is the most straight-forward in the whole process.
+Check that `README.md` has an updated compatibility matrix and that the
+url in the `kubectl` incantation (under "Installation") is updated to the
+new version you want to release.
+
+Now create the `kured-<release>-dockerhub.yaml`:
+
+```sh
+VERSION=1.3.0
+MANIFEST="kured-$VERSION-dockerhub.yaml"
+cat kured-rbac.yaml > "$MANIFEST"
+echo "---" >> "$MANIFEST"
+cat kured-ds.yaml >> "$MANIFEST"
+```
+
+The last thing you need to do is update the `image:` to point to the release
```

Comment:
Can be simply done with `sed -i`, but would require the 'no latest tag' command to be moved to e.g. above the `image` key.

```sh
sed "s#docker.io/weaveworks/kured#docker.io/weaveworks/kured:$VERSION#g" "kured-$VERSION-dockerhub.yaml"
```

Reply:
Nice one!